apps/task_test.go

```diff
@@ -41,15 +41,15 @@ func TestParseTasks(t *testing.T) {
 
 	expectedTasks := []Task{
 		{
-			ID:                 "test.47de43bd-1a81-11e5-bdb6-e6cb6734eaf8",
+			ID:                 "test.47de43bd-1a81-11e5-bdb6-e6cb6734eaf8._app.1",
```

Comment:
Will it be backwards compatible with Marathon <1.9?

Reply:
I've added an extra test, where the ids exactly match from event / task. See TestFindTaskByIdExactMatchBeforeMarathonVersionOneDotNine